fairlearn/reductions/resampling.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+import logging
+import math
+# import numpy as np
+# import pandas as pd
+# import scipy.optimize as opt
+# from sklearn import clone
+# from sklearn.dummy import DummyClassifier
+# from time import time
+# import inspect
+# import resampling
+
+
+logger = logging.getLogger(__name__)
+
+
+def resample_fixed_size_deterministic(X, y, weights):
```

Comment:
Can you add some tests of this function, to make it easier to see what's going on, please?